cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -375,11 +387,12 @@ func (uc *UpstreamController) updateNodeStatus(stop chan struct{}) {
 			case model.UpdateOperation:
```

Comment:
Can we add a create option here? So users can use a flag in edgecore to set whether to manually create node like kubelet.  ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_node_status.go#L370

Reply:
I was thinking we keep an option to allow auto registration of nodes at the edgecontroller side. If is it allowed, auto register all nodes that are connecting else reject all. Thoughts ?

Reply:
I think it should be decided by edgecore whether to auto-register instead of cloudcore

Reply:
Yes, we should do it in edgecore, consistent with kubelet

Reply:
Okay Sure, I will add the insert option in edge_core

Reply:
We already have code in edged. The messages were getting discarded at the cloud side.
https://github.com/kubeedge/kubeedge/blob/master/edge/pkg/edged/edged_status.go#L413

Reply:
I guess this change is sufficient or you guys will like me to add something more ?

Reply:
https://github.com/kubeedge/kubeedge/blob/master/edge/pkg/edged/edged_status.go#L413
L413 only register nodestatus in edged, we need a registerNode flag in edgecore. And move `createNode` func to edged. Can refer to the code of kubelet.

Reply:
We already have a function similar to what is there is kubelet for createNode in edged which creates a nodeStatus.
https://github.com/kubeedge/kubeedge/blob/master/edge/pkg/edged/edged_status.go#L50